tests/validation/tests/v3_api/common.py

```diff
@@ -22,7 +23,22 @@
                           "k8s_kube_config")
 MACHINE_TIMEOUT = float(os.environ.get('RANCHER_MACHINE_TIMEOUT', "1200"))
 
+TEST_OS = os.environ.get('RANCHER_TEST_OS', "linux")
 TEST_IMAGE = os.environ.get('RANCHER_TEST_IMAGE', "sangeetha/mytestcontainer")
+TEST_IMAGE_NGINX = os.environ.get('RANCHER_TEST_IMAGE_NGINX', "nginx")
+TEST_IMAGE_OS_BASE = os.environ.get('RANCHER_TEST_IMAGE_OS_BASE', "ubuntu")
+if TEST_OS == "windows":
+    DEFAULT_TIMEOUT = 300
+    if TEST_IMAGE == "sangeetha/mytestcontainer":
+        TEST_IMAGE = "maiwj/testcontainer:windows"
+    if TEST_IMAGE_NGINX == "nginx":
+        TEST_IMAGE_NGINX = "maiwj/testnginx:windows"
+    if TEST_IMAGE_OS_BASE == "ubuntu":
+        TEST_IMAGE_OS_BASE = "maiwj/testservercore:windows"
```

Comment:
Can we not have environment variables for the test images. And instead have GLOBAL variables which can be derived based on "TEST_OS" ?

Reply:
I think here could be reduced and use the passing variables during testing.

Reply:
Updated